config/services.go

```diff
@@ -246,6 +246,15 @@ func (s *AppServices) loadBux(ctx context.Context, appConfig *AppConfig, testMod
 		))
 	}
 
+	if appConfig.UseBeef {
+
+		if appConfig.Pulse == nil || appConfig.Pulse.PulseURL == "" || appConfig.Pulse.PulseAuthToken == "" {
```

Comment:
Actually PulseAuthToken shouldn't be required. Because you can deploy Pulse without authorization.